bucket/tachidesk.json

```diff
@@ -0,0 +1,31 @@
+{
+    "description": "A rewrite of Tachiyomi for the Desktop",
+    "homepage": "https://github.com/Suwayomi/Tachidesk",
+    "license": {
+        "identifier": "Mozilla Public License 2.0",
+        "url": "https://github.com/Suwayomi/Tachidesk/blob/master/LICENSE"
+    },
+    "url": "https://github.com/Suwayomi/Tachidesk/releases/download/v0.2.7/Tachidesk-v0.2.7-r312-win32.zip",
+    "hash": "3b97e60ffef0555a101a86cb3c5a6f7b86001d8dccc4861f77522b62c5ff8984",
+    "extract_dir": "Tachidesk-v0.2.7-r312-win32",
+    "version": "0.2.7",
+    "bin": [
+        [
+            "server.exe",
+            "tachidesk"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "server.exe",
+            "Tachidesk"
+        ]
+    ],
```

Comment:
Does not make sense to have shortcut for it as this is clearly meant to be executed as a service

```suggestion
```

Reply:
You are supposed to execute server.exe. That is the executable for Tachidesk which then opens Tachidesk on your browser. Its based of TachiWeb so they still call it server.exe.